### DISCUSSION TITLE
Missing House number field.

### DISCUSSION THREAD

**Discussion by minhdinh03** [woocommerce/woocommerce#51731] [Checkout Flow]
The checkout block doesn't contain house number field while it is required for further steps. I don't use any other plugin. How can i fix this

**Comment by mikejolley** [comment]
Hello @minhdinh03 

While we do not specifically label one of the fields "house number", this is what the `address 1` field is for. 

We avoid using specific terms because house number is not a term that applies to all countries or all address types (for example, a business address, a flat, or a house with a name rather than a number).

tl;dr the first address field is suitable for house number input.

I hope this clarifies things.

Thanks

**Reply by minhdinh03** [reply]
Hi Mike,

As i understand, user should type their street and house number in the "Street" field. But i still got the error that the house number is missing [See the attach image].
I tried reset the checkout page to make sure no field is accidentally deleted.

Thank you!
![image](https://github.com/user-attachments/assets/2657cb4a-8cf6-4e0c-a63c-23a94927d952)

**Reply by mikejolley** [reply]
I think this may be an extension (possibly shipping or payments?) adding custom validation and a custom field, but not integrating with the checkout block. If you can work out which plugin is doing this, you could then get in contact with the author.

**Comment by piezak** [comment]
Hello together,

@minhdinh03, @mikejolley, I face the similar issues. 
1. customer have send me a screenshot where the number was typed correctly and the form get back with identical message. (location:Poland, shipping: Poland). When I tried to recreate the issue was working fine, with the same data. We have cleared all possible things etc. did not help by the customer, one week later when she wants to buy again same issue. 
2. Sometimes I get some feedback about validation issues with email. When I tested it incognito with the same data is always fine. All is happening in incognito in different browsers.

**Reply by minhdinh03** [reply]
Hi @piezak,

I got this issue because we chose to use custom address field which it will collect street and house number as separated fields while using  block-layout checkout. Our delivery service's plugin (PostNL for Woocommerce) supports modify the address form that match requirement but only for shortcode version. So our solution were switch to shortcode checkout.
However, based on what you describe, i believe that you are facing a different problem. Because i got that bug in every test case (setting, browser). Our website sometimes behaves differently on different browser, too. I think you can try purging your cache.
